src/emu/debug/debugcmd.cpp

```diff
@@ -2429,6 +2429,8 @@ void debugger_commands::execute_dasm(int ref, int params, const char *param[])
 			}
 
 			/* disassemble the result */
+			disasm.clear();
+			disasm.seekp(0);
```

Comment:
this needs to be moved outside the `if` statement - the string was re-constructed each time through the loop, but now it won't be.